endpoints-support.md

```diff
@@ -0,0 +1,581 @@
+***Legend***
```

Comment:
Just a few details.
1- /rest/firmware-bundles```t```.
2- rest/firmware-drivers```/```. (Only to follow the pattern of others.).

Fixing these details, looks good for me.